src/include/mpi.h.in

```diff
@@ -998,6 +998,9 @@ typedef struct MPIX_Iov {
 #endif /* MPICH_SUPPRESS_PROTOTYPES */
 
 @HAVE_ROMIO@
+#ifdef HAVE_ROMIO
```

Comment:
I'm not sure I love putting this in a header the user will consume. The previous method seems preferable to that, honestly.

Reply:
We had an offline discussion. The conclusion is we'll always include mpi-io in mpi.h (and all bindings) and supply a mpi-io stub if `--without-romio` is configured.

Reply:
let's postpone the ROMIO binding generation. I have updated the fix to configure time autoconf substitution instead.